json/schemas/otp-schema.json

```diff
@@ -0,0 +1,50 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "title": "OTP Settings",
+    "description": "OTP Settings",
+    "type": "object",
+    "properties": {"$schema": {}},
+    "patternProperties": {
+        "^\\d{1,2}:\\d{1,2}$": {
+            "description": "Generic OTP Row",
+            "type": "object",
+            "properties": {
+                "ecc": {
+                    "description": "Protect with ECC",
+                    "type": "boolean"
+                },
+                "value": {
+                    "description": "Value to write",
+                    "type": ["array", "string", "integer"],
+                    "pattern": "^0x[0-9a-fA-F]{1,6}$",
+                    "items": {
```

Comment:
I guess `items` here could also be a `$ref`, but I'll let you decide if it's worth it or not.